### ISSUE TITLE
[Plugin] Jeager/Zipkin

### ISSUE THREAD

**Issue by jackskj** [backstage/backstage#716]
## Summary
Once developers find out that their application has an issue, it is very beneficial to see the application's traces. This can help identify the source of the issue.

## Motivation
I am using Istio service mesh (on top of k8s) along with Jeager tracing. 
I often depend on the tracing information to find the source of a particular issue or latency.
I believe that ability for developers to see traces of their application will be beneficial in diagnosing their application.

**Comment by OrkoHunter** [comment]
Re-opening since plugin suggestion issues are now exempt from [stalebot](https://github.com/backstage/backstage/blob/18f7345a6c29dcfccef7df081b1708f2bf8f62ff/.github/stale.yml#L9).

**Comment by djamaile** [comment]
Would be nice if there would be a complete istio plugin, which would show data from jaeger, kiali and the functionality a certain application is using from istio. Think about circuit breaking, fault injection and stuff.

**Comment by freben** [comment]
Yeah, those types of view when you get collected / correlated data from related systems gathered into one place, can be where the most value can be had for users.

**Comment by jbadeau** [comment]
We are actually builing an observability plugin which correlates traces with logs, and metrics. However, we did not set out to build a [dynnatrace,jaeger,...] clone as these tools do a perfectly good job of exploring traces. What we did is start from bottom up and watch for logged error or anomalies and then display these traces only. IF you need to do further exploring then you can follow a link to the source tool (i.e. hypertrace). We basically source logs, traces and metrics from different tools via kafka and normalize it into OpenTelemetetry models (Log,Trace,Metric) Then we display it in backstage.

Backstage has knowledge about your applications and can automatically filter for you. A simple example is shown here where the Resource name is added to the filter.

I am working on open sourcing this as soon as possible.

https://user-images.githubusercontent.com/152763/151332262-f7e88a49-e61c-4e96-920b-8f3244563a84.mp4

**Comment by ankitnayan** [comment]
What do you think about integrating [SigNoz](https://github.com/SigNoz/signoz)?

**Comment by dio** [comment]
cc. @tacigar

**Comment by mario-mui** [comment]
Are there something new? Do we have jaeger backstage plugin?
